src/query/graphql_checker.hpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <array>
+#include <string>
+#include <string_view>
+
+namespace memgraph::query {
+
+// TODO -1- (gvolfing) - check what happens if we can not provide
+// the same signature for any given functionality
+// TODO -2- (gvolfing) - test this once we have some facilities.
+// TODO -3- (gvolfing) - check if we should read in the mapping from
+// some files instead.
```

Comment:
We should, I would probably add a file Memgraph can read under `memgraph/config` folder, if something more advanced is needed, we can use `YAML` format and generate a specific "GraphQL config" file that you can easily parse here. The same pattern as it's used right now inside `glags.yaml` + `generate.py`. Ofc, in this case I think it doesn't have to be that advanced, it can be something simple, we just have to ensure the right config file ends up in the Memgraph packages (DEB & RPM).